be/src/olap/memtable.cpp

```diff
@@ -161,6 +155,15 @@ MemTable::~MemTable() {
             << std::endl
             << MemTracker::log_usage(_insert_mem_tracker->make_snapshot());
     DCHECK_EQ(_flush_mem_tracker->consumption(), 0);
+    _arena.reset();
+    _agg_buffer_pool.clear();
+    _vec_row_comparator.reset();
+    _input_mutable_block.clear_column_data();
```

Comment:
clear column data will not release memory. It just move end pointer.

Reply:
fixed